src/controllers/tasks/active.officers.controller.ts

```diff
@@ -66,7 +66,7 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
       const residentialAddress = formatAddressForDisplay(activeOfficerDetails?.residentialAddress);
       return res.render(Templates.ACTIVE_OFFICERS, {
         backLinkUrl: urlUtils.getUrlToPath(TASK_LIST_PATH, req),
-        officerErrorMsg: DIRECTOR_DETAILS_ERROR,
+        officerErrorMsg: OFFICER_DETAILS_ERROR,
```

Comment:
this controller is for the current live version with 1 director only. Do we know if that error message is supposed to change from director to officer for the current live version?

Reply:
Admittedly no, shall we just add a new error message for officers then and just leave the directors error as is?

Reply:
I would create new message, unless we have had a requirement to change the existing one then keep them separate